davtk/dap.py

```diff
@@ -12,7 +13,7 @@ def __init__(self, at_list, win_size, win_name=None, init_commands=None):
         # read settings from home dir and current dir
         settings = DavTKSettings()
         try:
-            parse_file(os.path.join(os.environ["HOME"],".daprc"), settings=settings)
+            parse_file(os.path.join(Path.home(),".daprc"), settings=settings)
```

Comment:
If we're going to use `Path` (which I approve of), can you check if `Path.home() / ".daprc"` or perhaps `str(Path.home() / '.daprc')` will work?  I'd prefer that, since it's cleaner than combining `os.path.join()` and `Path`.

Reply:
```suggestion
            parse_file(Path.home() / ".daprc", settings=settings)
```
Path.home() / ".daprc" does indeed work. I tested it on both Linux and Windows Python installs.